src/lib/ThumbnailsSidebar.js

```diff
@@ -311,6 +311,10 @@ class ThumbnailsSidebar {
      * @return {Promise} - promise reolves with the image HTMLElement or null if generation is in progress
      */
     createThumbnailImage(itemIndex) {
+        if (!this.thumbnailImageCache) {
+            return Promise.resolve(null);
```

Comment:
Should you create the cache or call to create the cache if it hasn't been created at this point?

Reply:
If it's destroyed already, we shouldn't create the cache again, right

Reply:
> Should you create the cache or call to create the cache if it hasn't been created at this point?

If it's destroyed already, we shouldn't create the cache again, right

Reply:
Why are we creating a thumbnail image AFTER the cache has been destroyed?